### PR TITLE
Fix SafepointUnifiedLogFileParser error logs produced by graalvm

### PR DESCRIPTION
**Pull request by arturowczarek**

### CODE REVIEW

File: io-utils/src/main/java/pl/ks/profiling/io/source/RegularFilesSource.java [hunk 1]
```diff
@@ -25,27 +25,6 @@ public String getName() {
         return this.name;
     }
 
-    @Override
```
Comment (arturowczarek) [inline io-utils/src/main/java/pl/ks/profiling/io/source/RegularFilesSource.java:28]: These were the same as in the parent class.

File: io-utils/src/test/groovy/pl/ks/profiling/io/FilesConcatenationSpec.groovy [hunk 2]
```diff
@@ -67,7 +67,7 @@ class FilesConcatenationSpec extends Specification {
 
     def "should concatenate files"() {
         when:
-        String concatenated = concatenateFiles("fileWithoutNewLine.log", "nextFile.log")
```
Comment (arturowczarek) [inline io-utils/src/test/groovy/pl/ks/profiling/io/FilesConcatenationSpec.groovy:70]: This didn't work at all.

File: io-utils/src/test/groovy/pl/ks/profiling/io/FilesConcatenationSpec.groovy [hunk 3]
```diff
@@ -76,7 +76,7 @@ some next file"""
 
     def "should keep new lines while concatenating"() {
         when:
-        String concatenated = concatenateFiles("fileWithNewLine.log", "nextFile.log")
```
Comment (arturowczarek) [inline io-utils/src/test/groovy/pl/ks/profiling/io/FilesConcatenationSpec.groovy:79]: Same here - didn't work at all.

File: io-utils/src/test/groovy/pl/ks/profiling/io/InputUtilsSpec.groovy [hunk 4]
```diff
@@ -27,7 +29,7 @@ class InputUtilsSpec extends Specification {
 
     def "should concatenate files as passed by default"() {
         given:
-        InputStream stream = InputUtils.getInputStream(files, null)
```
Comment (arturowczarek) [inline io-utils/src/test/groovy/pl/ks/profiling/io/InputUtilsSpec.groovy:30]: The invocations of this method lacked parameters and were used the wrong way.

File: io-utils/src/test/groovy/pl/ks/profiling/io/InputUtilsSpec.groovy [hunk 5]
```diff
@@ -85,14 +87,4 @@ class InputUtilsSpec extends Specification {
         return String.join("\n", new InputStreamReader(stream, StandardCharsets.UTF_8).readLines())
     }
 
-    BigDecimal getTimeStamp(String line) {
```
Comment (arturowczarek) [inline io-utils/src/test/groovy/pl/ks/profiling/io/InputUtilsSpec.groovy:88]: This wasn't used

File: pom.xml [hunk 6]
```diff
@@ -132,25 +132,15 @@
                     </execution>
                 </executions>
             </plugin>
-
             <plugin>
-                <groupId>org.codehaus.mojo</groupId>
```
Comment (arturowczarek) [inline pom.xml:137]: The tests didn't run at all.

File: safepoint-analyzer-commons/src/test/groovy/pl/ks/profiling/safepoint/analyzer/commons/shared/gc/parser/GCUnifiedLogFileParserSpec.groovy [hunk 7]
```diff
@@ -245,8 +245,8 @@ class GCUnifiedLogFileParserSpec extends Specification {
         gcEntry.regionsMax[REGIONS_SURVIVOR] == 184
         gcEntry.regionsMax[REGIONS_OLD] == null
         gcEntry.regionsMax[REGIONS_HUMONGOUS] == null
-        gcEntry.regionsSizeAfterGC == null
```
Comment (arturowczarek) [inline safepoint-analyzer-commons/src/test/groovy/pl/ks/profiling/safepoint/analyzer/commons/shared/gc/parser/GCUnifiedLogFileParserSpec.groovy:248]: I'm not sure if `[:]` is better than `Map.of()`. Let me know if you want me to change it.

File: safepoint-analyzer-commons/src/test/groovy/pl/ks/profiling/safepoint/analyzer/commons/shared/safepoint/parser/SafepointUnifiedLogFileParserTest.groovy [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+        entry.stoppedTime == 0.000107250
+        entry.completed
+    }
+
+    def "should should parse graal log properly"() {
+        given:
+        SafepointUnifiedLogFileParser parser = new SafepointUnifiedLogFileParser()
+
+        when:
+        parser.parseLine("[2023-07-17T08:20:21.129+0000][5086.354s][info ][safepoint      ] Safepoint \"G1CollectForAllocation\", Time since last: 5972932511 ns, Reaching safepoint: 108298 ns, Cleanup: 83917 ns, At safepoint: 113194973 ns, Total: 113387188 ns")
```
Comment (arturowczarek) [inline safepoint-analyzer-commons/src/test/groovy/pl/ks/profiling/safepoint/analyzer/commons/shared/safepoint/parser/SafepointUnifiedLogFileParserTest.groovy:30]: This was the line which brought on the need for this change. It comes from graal vm and raised the exception.

File: safepoint-analyzer-web/src/main/java/pl/ks/profiling/safepoint/analyzer/web/StatsController.java [hunk 9]
```diff
@@ -30,7 +30,7 @@
 import pl.ks.profiling.safepoint.analyzer.commons.shared.report.JvmLogFile;
 import pl.ks.profiling.web.commons.WelcomePage;
 
-import javax.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletRequest;
```
Comment (arturowczarek) [inline safepoint-analyzer-web/src/main/java/pl/ks/profiling/safepoint/analyzer/web/StatsController.java:33]: `javax` is no longer used

File: safepoint-analyzer-commons/src/test/groovy/pl/ks/profiling/safepoint/analyzer/commons/shared/gc/parser/GCUnifiedLogFileParserSpec.groovy (hunk 7 above)
Comment (krzysztofslusarski) [reply safepoint-analyzer-commons/src/test/groovy/pl/ks/profiling/safepoint/analyzer/commons/shared/gc/parser/GCUnifiedLogFileParserSpec.groovy:248]: ```Collections.emptyMap()```?

**Review by krzysztofslusarski — APPROVED** [review]

**Pull request merged**